exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/ParquetComparisonPredicates.java

```diff
@@ -30,8 +30,9 @@
  * Comparison predicates for parquet filter pushdown.
  */
 public class ParquetComparisonPredicates {
```

Comment:
There is too much code duplicated in the original implementation and it needs to be refactored. Please see PR #1259.

Reply:
Hello VRozov,
you're right. I though less code modification was better (my preceding team was afraid of code refactoring...). I have refactored the class.
best regards